Ribbon for RC and unsupported

lzap opened the pull request:
This add ribon "RC" and "Unsupported" according to URL. This is hacky, but it allows us to avoid forking HTML/CSS template. Unfortunately, asciidoc syntax (and attributes) cannot be used in this header "snippet", ideally, this should have been an attribute.

Fixes #695

lzap commented:
@ekohl the ribbon named `WIP!` is now shown for all guides you marked as incorrect, it is clickable and gets you to our README which I elaborated and updated.

lzap commented on guides/common/docinfo.html:
```diff
@@ -1,8 +1,33 @@
+  rl.includes("/Upgrading_and_Updating/index-foreman-el.html" ||
+  rl.includes("/Upgrading_and_Updating/index-foreman-deb.html" ||
+  rl.includes("/Configuring_Load_Balancer/index-foreman-el.html" ||
+  rl.includes("/Configuring_Load_Balancer/index-foreman-deb.html" {
```
This could be a regexp, but I specifically want this to be just a list so we can easily be deleting guides which are considered finished.

maximiliankolb replied:
I guess this is fine for now IMHO.

ekohl commented:
> @ekohl the ribbon named `WIP!` is now shown for all guides you marked as incorrect, it is clickable and gets you to our README which I elaborated and updated.

WIP is an abbreviation. Should we use a common English word like Draft?

lzap commented:
Absolutely, draft looks better. Rebased. HTML links will fail, working on that.

lzap commented:
If we could land this before 3.1 branching that would be awesome, thanks! Maybe @maximiliankolb could help since @ekohl is busy with getting the bits out.

maximiliankolb commented on README.md:
```suggestion
This repository and `docs.theforeman.org` site contains work in progress content.
Some guides, specifically those marked with a `DRAFT` ribbon in the top-right corner, may contain irrelevant or incorrect information.
If you see such guide, please visit the official [Foreman documentation](https://theforeman.org/documentation.html).
Feel free to open a PR or issue on Github.
```

maximiliankolb reviewed (commented):
I could not find a version with the "DRAFT" ribbon. Can you please share a link or screenshot?

lzap commented on guides/common/docinfo.html:
```diff
@@ -1,8 +1,33 @@
 <script>
 const rl = document.location.toString();
+ifdef::ribbon[]
```
This line was breaking JS, not sure what I meant by this. Removing.

lzap commented:
And there were ton of JS syntax errors, fixed them all now it shows up properly. Apologies for filing untested code. I also updated numbers: 3.1 and 3.0 are supported now, 3.2 will be RC, drafts are from the given list.
